README.md

```diff
@@ -2,111 +2,78 @@
+#### Schema migrations and initial Admin user.
+The second time that stash-box is run, stash-box will run the schema migrations to create the required tables. It will also generate a `root` user with a random password and an API key. These credentials are printed once to stdout and are not logged. The system will regenerate the root user on startup if it does not exist. You can force the system to create a new root user by deleting the root user row from the database and restarting Stash-box. You'll need to capture the console output with your Admin user on the first successful StashDB executable start. Otherwise, you will need to allow Postgres to re-create the database before it will re-post a new `root` user.
 
-After ensuring the database connection string is correct and the database server is available, the stash-box executable may be rerun.
+# Stash-box CLI and Configuration
 
-The second time that stash-box is run, stash-box will run the schema migrations to create the required tables. It will also generate a `root` user with a random password and an API key. These credentials are printed once to stdout and are not logged. The root user will be regenerated on startup if it does not exist, so a new root user may be created by deleting the root user row from the database and restarting stash-box.
+Stash-box is a tool with command line options to make it easier. To see what options are available, run `stash-box --help` in your terminal.
 
-## CLI
+Here's an example of how you can run stash-box locally on port 80:
 
-Stash-box provides some command line options.  See what is currently available by running `stash-box --help`.
+`stash-box --host 127.0.0.1 --port 80`
 
-For example, to run stash locally on port 80 run it like this (OSX / Linux) `stash-box --host 127.0.0.1 --port 80`.
 
-## Configuration
+**Note:** This command should work on OSX / Linux.
 
-Stash-box generates a configuration file `stash-box-config.yml` in the current working directory when it is first started up. This configuration file is generated with the following defaults:
-- running on `0.0.0.0` port `9998`
+When you start stash-box for the first time, it generates a configuration file called `stash-box-config.yml` in your current working directory. This file contains default settings for stash-box, including:
 
-The graphql playground and cross-domain cookies can be disabled by setting `is_production: true`.
+- Host: `0.0.0.0`
+- Port: `9998`
 
-### API keys and authorisation
+You can change these defaults if needed. For example, if you want to disable the Graphql playground and cross-domain cookies, you can set `is_production` to `true`.
 
-A user may be authenticated in one of two ways. Session-based management is possible by logging in via `/login`, passing form values for `username` and `password` in plain text. This sets a cookie which is required for subsequent requests. The session can be ended with a request to `/logout`.
+## API Keys and Authorization
```

Comment:
Would change it to `## API keys and authorization` to keep capitalization consistent with other headings.